docs/user-guide/projects/project-settings.md

```diff
@@ -199,16 +199,16 @@ Trigger when a source document is added or removed from project version
 #### Manual event
 Trigger by user manually.
 
-To trigger such event, first project maintainer will need to add and enable the webhook.
-Then translator can select the language he/she has access to from the version page, then from the top right dropdown menu, you should be able to fire the event to registered webhook endpoint.
+To trigger such event, first, the project maintainer will need to add and enable the webhook.
+Then the translator can select the language he/she has access to from the languages tab on the version overview page, then from the top right dropdown menu of the documents column, he should be able to fire the event to the registered webhook endpoint.
```

Comment:
Substitute _he/she has_ for _they have_, and _he should be able to_ for _they can_
I think there should be a period after "overview page".